docs/pages/project/contributing/contributing-docs.md

```diff
@@ -401,5 +383,38 @@ The assign tag is used to create a new variable. It is written in the following
 {% capture code_content %}{{ "{% assign variable1 = true " }}%}{% endcapture %}
 {% include code.html code=code_content %}
 
+### Alert
```

Comment:
Change this heading to `alert.html` and add it after the `Alerts` section, in order to maintain consistency.